doc/src/devdocs/callconv.md

```diff
@@ -1,60 +1,43 @@
-.. _devdocs-callconv:
-
-*******************
-Calling Conventions
-*******************
-
-.. currentmodule:: Base
+# Calling Conventions
 
 Julia uses three calling conventions for four distinct purposes:
 
-+-------+--------------+----------------------------------+
-|Name   |Prefix        |Purpose                           |
-+=======+==============+==================================+
-|Native |``julia_``    | Speed via specialized signatures |
-+-------+--------------+----------------------------------+
-|JL Call|``jlcall_``   | Wrapper for generic calls        |
-+       +--------------+----------------------------------+
-|       |``jl_``       | Builtins                         |
-+-------+--------------+----------------------------------+
-|C ABI  |``jlcapi_``   | Wrapper callable from C          |
-+-------+--------------+----------------------------------+
+| Name    | Prefix    | Purpose                          |
+|:------- |:--------- |:-------------------------------- |
+| Native  | `julia_`  | Speed via specialized signatures |
+| JL Call | `jlcall_` | Wrapper for generic calls        |
+| JL Call | `jl_`     | Builtins                         |
```

Comment:
Nope, markdown doesn't usually have that I think, though there are some dialects, such as pandoc, that do seem to support those kind of things, so we could probably just extend our parser as well.